### DISCUSSION TITLE
Fan-made Mint-Y icons

### DISCUSSION THREAD

**Discussion by mtunixic** [linuxmint/discussions#602] [Ideas]
> Heya! I'm a big fan of the **Mint-Y** icon theme, and I've made some custom ones myself!
> _Is there any chance for these to get officially added?_

![image](https://github.com/user-attachments/assets/b2154e28-9fda-4166-818f-46f49c638a72) ![image](https://github.com/user-attachments/assets/2714b4ca-393e-457a-9c5a-4d5ace483fde) ![image](https://github.com/user-attachments/assets/89506984-2395-4c69-aa8e-c42e0140ba50) ![image](https://github.com/user-attachments/assets/f8ff6989-0beb-4732-9318-6df4f5ed0c30)
![image](https://github.com/user-attachments/assets/d7798865-c1ef-4eef-b007-8e347369c1f0) ![image](https://github.com/user-attachments/assets/0e960fc9-75d3-49fe-8fd3-7ca81271c1ce)
> **_Thank you for your attention :)_**

**Comment by anaximeno** [comment]
you'll have to submit them to https://github.com/linuxmint/mint-y-icons
head out to https://matrix.to/#/#linuxmint-artists:matrix.org if you need help (there are some people that have contributed with icons for mint-y and might be able to help you)
